css/style.css

```diff
@@ -1,32 +1,41 @@
-/**
- * Beware that Qt Assistant (QTextBrowser) supports only a limited set of 
- * HTML/CSS: http://qt-project.org/doc/qt-5/richtext-html-subset.html
- */
```

Comment:
This comment is quite important at the moment and must not be removed.

Maybe the CSS needs to be split in a part which is per publishing path (online vs. offline), and a part which is shared.